ddtrace/contrib/pytest/plugin.py

```diff
@@ -46,6 +47,17 @@ def inner_encode(obj):
     return json.dumps(params, default=inner_encode)
 
 
+def _extract_repository_name(repository_url):
+    # type: (str) -> str
+    """Extract repository name from repository url."""
+    try:
+        parsed_path = compat.parse.urlparse(repository_url).path
+        return parsed_path.split("/")[-1].split(".git")[0]
+    except Exception:
+        # In case of parsing error, default to repository url
```

Comment:
Worth logging a warning? Would help with debugging if some unexpected value is provided.